docs/nidcpower/rep_caps.rst

```diff
@@ -42,9 +50,14 @@ instruments
 
         .. code:: python
 
-            session.instruments['0-2'].channel_enabled = True
+            print(session.instruments['Dev1'].serial_number)
+
+        prints :py:attr:`serial_number` for instruments 'Dev1'.
+
+        .. code:: python
 
-        passes a string of :python:`'0, 1, 2'` to the set attribute function.
+            print(session.instruments['Dev1', 'Dev2', '3rdDevice'].serial_number)
 
+        prints :py:attr:`serial_number` for instruments 'Dev1', 'Dev2', '3rdDevice' or errors if the value is not the same for all.
```

Comment:
I included the part about the error behavior on a whim. It should hold true for any multi-capability property read, but let me know if you think I should leave it out.

Reply:
Whether the driver allows passing multiple rep caps or not varies. The behavior you describe for serial_number happens to be accurate for serial_number but no guaranteee for other things. I think it's wiser to leave out.

Reply:
I was actually referring to the inability to retrieve a property value for multiple capabilities if the value is not the same for all.

Reply:
I understand. I'm saying that the ability to retrieve a property value for multiple capabilities if the value is the same for all is not a universally true thing. It can vary from attribute to attribute. It just happens that we implemented the described behavior for _most_ attributes in our driver runtimes.